src/l10n.json

```diff
@@ -62,6 +62,7 @@
     "general.scratchJr": "ScratchJr",
     "general.scratchStore": "Scratch Store",
     "general.search": "Search",
+    "general.searchEmpty": "Nothing found",
```

Comment:
@thisandagain @christanbalch @ntlrsk thoughts on this language for an empty search page?

Reply:
@thisandagain @christanbalch @ntlrsk just wanted to check one more time about empty search language – does this look alright to you?

Reply:
This looks fine for now. Thanks @mewtaylor